deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -2026,10 +2026,7 @@ class ChatViewController: UITableViewController, UITableViewDropDelegate {
     }
 
     private func copyToClipboard(ids: [Int]) {
-        let pasteboard = UIPasteboard.general
-        pasteboard.string = ""
```

Comment:
I remember setting string to blank to avoid some crash in https://github.com/deltachat/deltachat-ios/commit/f2d987ae29ad84add427f46cdd9a4253853b9f2e, but maybe it's not necessary when the code is organized this way.

Reply:
good pointer. however, i think that time the bug could also have been fixed by removing the line.

we're now building the string and after that touching the clipboard - this is what we're doing at other places as well, so i think, that should be okay.